.github/workflows/docker-publish.yml

```
@@ -1,96 +1,78 @@
 name: docker
 
 on:
-    push:
-      tags:
-        - "v*.*.*"
-    schedule:
-      - cron: "0 0 * * *"
-    # Trigger without any parameters a proactive rebuild
-    workflow_dispatch: {}
-    workflow_call:
+  push:
+    tags:
+      - "v*.*.*"
+  schedule:
+    - cron: "0 0 * * *"
+  # Trigger without any parameters a proactive rebuild
+  workflow_dispatch: {}
+  workflow_call:
 
 env:
-    REGISTRY: ghcr.io
-    # Will resolve to foundry-rs/foundry
-    IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+  # Will resolve to foundry-rs/foundry
+  IMAGE_NAME: ${{ github.repository }}
 
 jobs:
-    container:
-        runs-on: Linux-20.04
-        # https://docs.github.com/en/actions/reference/authentication-in-a-workflow
-        permissions:
-            id-token: write
-            packages: write
-            contents: read
-        timeout-minutes: 120
-        steps:
-            - name: Checkout repository
-              id: checkout
-              uses: actions/checkout@v4
+  build:
+    name: build and push
+    runs-on: ubuntu-20.04
```

We have faster custom runners

```suggestion
    runs-on: Linux-20.04
```